minimesos/src/main/java/com/containersol/minimesos/mesos/MesosContainerImpl.java

```diff
@@ -51,13 +49,7 @@ public String getImageTag() {
         return imageTag;
     }
 
-    protected String[] createMesosLocalEnvironment() {
-        envVars.putAll(getDefaultEnvVars());
-        envVars.putAll(getSharedEnvVars());
-        return createEnvironment();
-    }
-
-    protected Map<String, String> getSharedEnvVars() {
+    protected final Map<String, String> getSharedEnvVars() {
```

Comment:
It reflects the new strategy of the subclasses creating the final list of environment variables. As the shared variables are still needed they should only be provided by the superclass to the subclass. There is no reason to override the method as a subclass can just choose not to use it.